errors.go

```diff
@@ -165,8 +176,27 @@ type ErrorResponse struct {
 
 const defaultErrorCT = "text/html; charset=utf-8"
 
-func defaultErrorHandler(status int, origErr error, c Context) error {
+type stackTracer interface {
+	StackTrace() pkgError.StackTrace
+}
+
+// Attempt to extract stacktrace error with stack if it implements stacktracer
+func getTrace(err error) string {
+
+	if errStack, ok := err.(stackTracer); ok {
+		return fmt.Sprintf("%+v", errStack.StackTrace())
+	} else {
+		return err.Error()
+	}
+}
+
+func isUnsafeEnvironment(c Context) bool {
 	env := c.Value("env")
+	return env != nil && env.(string) != "development" && env.(string) != "test"
```

Comment:
the environment "test" should execute the same code set as "production" (with data for testing) and this is not acceptable.